### DISCUSSION TITLE
Which block style should we use for tips in the migration guide?

### DISCUSSION THREAD

**Discussion by peter-csala** [App-vNext/Polly#1766] [General]
I've just realized that the `[!IMPORTANT]` blocks are [rendered](https://dotnet.github.io/docfx/docs/markdown.html?tabs=linux%2Cdotnet#alerts) as red boxes via docfx:

DOCFX
![Screenshot 2023-11-02 at 10 51 13](https://github.com/App-vNext/Polly/assets/57183693/26a3addd-f5be-480d-8373-2afcecb17fdd)

GITHUB
![Screenshot 2023-11-02 at 10 51 27](https://github.com/App-vNext/Polly/assets/57183693/830bf3c6-0e9e-4d7b-ab91-96f6362294df)

It feels a bit "aggressive" to me for a tip. When I changed to `[!TIP]` then github rendering failed :(

DOCFX
![Screenshot 2023-11-02 at 10 49 46](https://github.com/App-vNext/Polly/assets/57183693/7e3c61e7-57f6-45bf-81dc-5d3e7ddf716d)

GITHUB
![Screenshot 2023-11-02 at 10 49 39](https://github.com/App-vNext/Polly/assets/57183693/bc48f053-58ac-4529-b637-3f7e4e12c984)

What do you guys think what should we use on the migration V8 page?

I've found this bit odd [alternative](https://stackoverflow.com/a/76515821/13268855) to render tip as a single column table. 

cc: @martincostello , @martintmk

**Comment by martintmk** [comment]
I think we can get away with TIP being rendered only on pollydocs.org as that's how this content is consumed anyway.

Otherwise, let's not do anything special and use INFORMATION or NOTE instead.

**Reply by martincostello** [reply]
I agree - out of the two let's use TIP and we'll just have to live with the fact they don't render in GitHub.
